dotCMS/src/main/webapp/html/js/tinymce/js/tinymce/plugins/dotimageclipboard/dialog.jsp

```diff
@@ -33,13 +33,21 @@ if(clipboard ==null){
 
 <style>
 
+body {
+	margin: 0;
```

Comment:
We need to make this to use Roboto from the instance of dotcms, see my comment in the PR.

Reply:
updated